cmd2.py

```diff
@@ -1589,32 +1612,28 @@ def quit():
             self._in_py = False
         return self._should_quit
 
-    # noinspection PyUnusedLocal
-    @options([], arg_desc='<script_path> [script_arguments]')
-    def do_pyscript(self, arg, opts=None):
+    @with_argument_list
+    def do_pyscript(self, arglist):
```

Comment:
For some of these commands that got converted from an empty **options** decorator with a non-empty ``arg_desc`` argument to a **with_argument_list** decorator, we may want to add an explicit "usage" statement to the help text.

Reply:
Thanks for fixing the unit tests that I broke 😳